LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright [2022] [Hamburg Bit-Bots]
```

Comment:
```suggestion
   Copyright [yyyy] [name of copyright owner]
```

This one can be left as is. (See: https://github.com/ros2/rclcpp/blob/master/LICENSE)